go/enclave/rollupchain/rollup_chain.go

```diff
@@ -12,28 +12,28 @@ import (
 	"sync"
 
 	"github.com/ethereum/go-ethereum/common/hexutil"
-	"github.com/ethereum/go-ethereum/core"
+	"github.com/obscuronet/go-obscuro/go/common/gethapi"
```

Comment:
Can the imports be ordered ? Following https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering